bin/5.0.x-dev/prepare_project_edition.sh

```diff
@@ -180,4 +180,88 @@ fi
 echo '> Generate GraphQL schema'
 docker compose --env-file=.env exec -T --user www-data app sh -c "php bin/console ibexa:graphql:generate-schema"
 
+########################################################################################################################
+
+# Delete old GraphQL schema
+docker compose --env-file=.env exec -T --user www-data app sh -c "rm -r config/graphql"
+
+# Remove Stimulus bootstrap
+docker compose --env-file=.env exec -T --user www-data app sh -c "rm assets/bootstrap.js"
+docker compose --env-file=.env exec -T --user www-data app sh -c "sed -i '/bootstrap/d' assets/app.js"
+docker compose --env-file=.env exec -T --user www-data app sh -c "sed -i '/start the Stimulus application/d' assets/app.js"
+
+# Move from annotation to attribute
+#docker compose sed -i 's/type: annotation/type: attribute/g' config/routes/annotations.yaml
+docker compose --env-file=.env exec -T --user www-data app sh -c "rm config/routes/annotations.yaml"
+docker compose --env-file=.env exec -T --user www-data app sh -c "rm config/routes.yaml"
+
+# TMP: Accept dev version (5.0.x-dev)
+docker compose --env-file=.env exec -T --user www-data app sh -c "composer config minimum-stability dev"
+
+# Update required PHP version
+docker compose --env-file=.env exec -T --user www-data app sh -c "composer require --no-update 'php:>=8.3'"
+# Update required Symfony version
+docker compose --env-file=.env exec -T --user www-data app sh -c "composer config extra.symfony.require '7.2.*'"
+
+
+# Upgrade Ibexa and Symfony packages (main app)
+docker compose --env-file=.env exec -T --user www-data app sh -c "composer require --no-update \
+  ibexa/commerce:5.0.x-dev \
```

Comment:
```suggestion
  ibexa/commerce:5.0.x-dev \
  ibexa/behat:5.0.x-dev \
  ibexa/docker:5.0.x-dev \
```
We will probably need to update these as well